libs/books/data-access/src/lib/+state/reading-list.reducer.spec.ts

```diff
@@ -51,6 +51,32 @@ describe('Books Reducer', () => {
 
       expect(result.ids).toEqual(['A', 'B', 'C']);
     });
+
+    it('confirmedMarkAsFinished should display finished as true', () =>{
+      const readingList = {
```

Comment:
Instead of creating same object multiple times, try to create generic function which returns dynamic object.